src/api/example/controllers/example-find-all-controller.js

```diff
@@ -1,6 +1,16 @@
 import { findAllExampleData } from '~/src/api/example/helpers/find-all-example-data.js'
 
+/**
+ * Example controller: remove as needed
```

Comment:
I'd probably remove the `: remove as needed` - this is implicit with it being an example controller